tools/varint/src/main/java/org/apache/polaris/tools/varint/VarInt.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.tools.varint;
```

Comment:
I'm mainly concerned about having these in a new module and in the o.a.polaris.tools namespace. If this is for enabling the connector for Mongo shouldn't it live in that namespace? Otherwise we have to consider the general utility to the entire project and the burden we take on by exposing these public classes and apis? If we did use an already existing library or placed it in a nessie/mongo specific package that wouldn't be required